config/data_sources/reshare/01_patron_requests.yml

```diff
@@ -9,6 +9,7 @@ column_mappings:
   "pr_date_created": pr_date_created
   "pr_pub_date": pr_pub_date
   "pr_edition": pr_edition
+  "__start": __start
```

Comment:
When we ingest via CSV, any headers like __start will be automatically converted to "start".  So having a column that starts with not a letter will cause issues down the line.